Libplanet.Store.Remote/Libplanet.Store.Remote.csproj

```diff
@@ -0,0 +1,35 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
```

Comment:
`Grpc.AspNetCore` supports only net6.0 or greater, so `netstandard2.1` can not be used.
https://www.nuget.org/packages/Grpc.AspNetCore#supportedframeworks-body-tab

Reply:
@s2quake right.

Reply:
If this project only supports `net6.0`, it would be nice to add `net8.0` and use language version `12`. In the age of `net9.0`, isn't `net6.0` a relic? We need to go one step further.

Reply:
> it would be nice to add net8.0 and use language version 12.

The target of this service is a project which uses `Libplanet`. It means most projects use `net6.0` and language `8`. Therefore, we cannot bump the version immediately. @s2quake

Reply:
Additionally, I tried bumping up to `net8.0` and lang version `12` to [`NineChronicles.Headless.`](https://github.com/planetarium/NineChronicles.Headless) But it causes a build error rn. I don't wanna bumping up for using this service. It should be another PR or task.